src/components/edit/RecentChangeHistory.tsx

```diff
@@ -68,27 +68,31 @@ export const ChangesetCard: React.FC<ChangsetRowProps> = ({ changeset }) => {
 
   )
 }
-
-const ClimbChange = ({ changeId, fullDocument, updateDescription, dbOp }: ChangeType): JSX.Element | null => {
+const ClimbChange = ({ changeId, fullDocument, updateDescription, dbOp }: ChangeType): JSX.Element => {
   if (fullDocument.__typename !== DocumentTypeName.Climb) {
-    return null
+    return <></>
```

Comment:
Why would we want to return an empty fragment vs null?  
https://react.dev/learn/conditional-rendering#conditionally-returning-nothing-with-null